scripts/get-sampleapp-url.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+#########################################################################################################
+# Script to retrieve the websphere sample app url from the openshift routes in the sample app namespace
+#########################################################################################################
+
+import json
+import logging
+import os
+import subprocess
+import sys
+import time
+
+input = sys.stdin.read()
+input_json = json.loads(input)
+
+# json response format
+json_response = {"response": ""}
+
+# debugging file init
+DEBUGFILE = "/tmp/getsampleappurl.log"
```

Comment:
let's just use `getsampleappurl.log` instead of `/tmp/getsampleappurl.log` That might fail on Windows. We could generate a path that would work on both but I think putting it in the PWD is fine

Reply:
fixed